SRC/KEYB.CPP

```diff
@@ -0,0 +1,144 @@
+#include <set>
+#include <cstdint>
+#include <SDL.h>
+#include "KEYB.H"
+
+namespace tk_port
+{
+namespace keyb
+{
+
+const uint32_t KEY_REPEAT_TIMER_CODE = 43615681;  // Random number heh heh
+const uint32_t SYMBOL_KEY_REPEAT_START = 500;     // ms
```

Comment:
Repeat start delay is considerably shorter on DOSBox. Maybe half or so.

Reply:
I tried to search around for how the key repeat is actually implemented (i.e. does it happen in the window system, in the keyboard driver, somewhere else in the OS, or in the keyboard itself) but didn't find much info. My point is whether it can depend on user configuration or not.

Do you think we should make the initial interval faster?

Reply:
Feel free to go ahead with this. It can be tweaked later if needed.